starfyre/compiler.py

```diff
@@ -2,15 +2,30 @@
 from pathlib import Path
 
 
-def get_fyre_files(project_dir):
+def get_fyre_files(project_dir):   
     fyre_files = []
     for file in os.listdir(project_dir):
         if file.endswith(".fyre"):
             fyre_files.append(file)
     return fyre_files
 
+def resolve_css_import(import_statement, working_directory):
+    """Read a css file and save it's content to a list"""
+    css_content = []    
+    if "" in import_statement:
+        css_file_name = import_statement.replace('"', "'")        
+    css_file_name = css_file_name.split("'")[1]   
```

Comment:
Hey @SuelenKarbivnychyy , I love the approach on line 16! Very smart!

However, line 17 still has a bug. This statement would still work on `import 'test.css`, which should not be the case. Using `split` will require us to handle a lot of edge cases.

Reply:
Thank you again. I've addressed that out. :)